### PR TITLE
Improve e2e test suite

### PR DESCRIPTION
**Pull request by nox1134**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🟢
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 

## Description
(This is in continuation of issue #13884.)
1. While running the Codeception tests, there were deprecation warnings that caused the command to fail, even though the test was passing. I made some changes in codeception.yml to silence those warnings.
2. Updated the batch delete contact tests to include dynamic waits, as the test was sometimes failing when the changes didn’t load in time.
3. Updated the change owner test to include a dynamic wait.
4. Updated the sample contacts CSV file to include some new contacts.
5. Moved the import CSV test to the end.

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Edit .env.local:
```
# .env.local
APP_ENV=test
APP_DEBUG=1 
```
3. `ddev start`
4. `ddev ssh`
5. Build the test dependencies:
`bin/codecept build`
6. `bin/codecept run acceptance ContactManagementCest `
7. Ensure that all tests pass and there are no depreciation messages.

For more details, follow this guide for running the tests: https://devdocs.mautic.org/en/5.x/testing/e2e_test_suite.html.
If you encounter any issues, try running `composer update` once.

### CODE REVIEW

**Review by shinde-rahul — APPROVED** [review]
LGTM!!!

File: tests/_data/10contacts.csv
Comment (RCheesley) [inline tests/_data/10contacts.csv:1]: Just a general point, in the future let's use more diverse names in our test data. See the Google Style Guide for examples:

https://developers.google.com/style/examples

**Review by RCheesley — DISMISSED** [review]
Great work!

I had a few issues as I hadn't realised the env files needed updating and that I needed to run `composer update` in my local environment.

Once I fixed those (thanks @nox1134 for the assist!) the tests run fine and there aren't any deprecation warnings.

I added a note about using more diverse names in our sample data, this is not mandatory but a resource to consider in the future when creating sample data. It doesn't block this PR unless you particularly want to address it :)

File: tests/_data/10contacts.csv
Comment (nox1134) [reply tests/_data/10contacts.csv:1]: Okay! I'll update the CSV files with more diverse names.

**Pull request merged**